front-end/src/app/reports/transactions/double-transaction-detail/double-transaction-detail.component.html

```diff
@@ -64,8 +64,44 @@ <h3>{{ templateMap.amountInputHeader }}</h3>
             [memoCodeReadOnly]="isMemoCodeReadOnly(transaction?.transactionType)"
             [negativeAmountValueOnly]="!!transaction?.transactionType?.negativeAmountValueOnly"
             [showAggregate]="!!transaction?.transactionType?.showAggregate"
+            [transaction]="transaction"
           ></app-amount-input>
           <p-divider></p-divider>
+          <ng-container *ngIf="this.hasParentCandidateInformationInput">
+            <h3>{{ templateMap['candidateInputHeader'] }}</h3>
+            <app-committee-input
+              [form]="form"
+              [formSubmitted]="formSubmitted"
+              [templateMap]="templateMap"
+              entityRole="COMMITTEE"
+              [includeFecId]="true"
+              [readonly]="true"
+            ></app-committee-input>
+            <app-transaction-contact-lookup
+              [contactTypeFormControl]="candidateContactTypeFormControl"
+              [contactTypeOptions]="candidateContactTypeOption"
+              [contactTypeReadOnly]="true"
+              (contactSelect)="onSecondaryContactLookupSelect($event)"
+            ></app-transaction-contact-lookup>
+            <ng-container *ngIf="transaction?.contact_2">
+              <app-candidate-input
+                [form]="form"
+                [formSubmitted]="formSubmitted"
+                [templateMap]="templateMap"
+              ></app-candidate-input>
+            </ng-container>
+            <p-divider></p-divider>
+          </ng-container>
+          <ng-container *ngIf="hasParentElectionInformationInput">
+            <h3>Election Information</h3>
+            <app-election-input
+              [form]="form"
+              [formSubmitted]="formSubmitted"
+              [templateMap]="templateMap"
+            ></app-election-input>
+            <p-divider></p-divider>
+          </ng-container>
+          <p-divider></p-divider>
```

Comment:
May want to move this divider inside the ng-container above since it's getting a double divider in the ui

Reply:
:+1: